test/jdk/sun/security/pkcs11/Secmod/AddPrivateKey.java

```diff
@@ -68,6 +68,10 @@ public static void main(String[] args) throws Exception {
                     BASE + File.separator + args[1]);
         }
 
+        double version = getNSSVersion();
+        if (version == 0.0 || version >= 3.55) {
+            useSqlite(true);
+        }
```

Comment:
Instead of updating various tests with this block, how about doing this inside SecmodTest.initSecmod()?

Reply:
Such a change would apply NSS version number checks to `TestNssDbSqlite` which currently doesn't have them.  But wait! If `TestNssDbSqlite` has been running for years on all NSS versions, then it should be safe to make sqlite the default for all tests.

Reply:
Yes, for new NSS releases, it should be safe to set sqlite to true by default. The tests which relies on sqlite like `TestNssDbSqlite` can always call useSqlite(true) regardless of default.

Reply:
@valeriepeng Thanks for the review! I'll set sqlite to true by default and retest.